src/promise-katas-1.js

```diff
@@ -33,71 +33,125 @@ Remember, a Promise can only resolve successfully, or reject when it fails to re
 // when we call returnPromise like 'returnPromise()' we expect our function to use the 
 // 'resolve' Promise callback function in order to resolve successfully
 
-const returnPromise = () => { };
+const returnPromise = () => {
+    return new Promise((resolve, _) => {
+        resolve()
+    })
+ };
 
 // 2 Create a function that returns a promise which returns 10 if it resolves succesfully
 // Example:
 // returnTen() resolves to the value 10
 
-const returnTen = () => { };
+const returnTen = () => { 
+    return new Promise((resolve, _) => {
+        resolve(10)
+    })
+};
 
 // 3 Create a function that returns a promise which returns 'string' if it resolves succesfully
 // Example:
 // returnString() resolves to the value 'string'
 
-const returnString = () => { };
+const returnString = () => { 
+    return new Promise((resolve, _) => {
+        resolve('string')
+    })
+};
 
 // 4 Create a function that returns a promise which returns an object with a key of name and value of 'Bob' if it resolves succesfully
 // Example:
 // returnBob() resolves to the value { name: 'Bob' }
 
-const returnBob = () => { };
+const returnBob = () => { 
+    return new Promise((resolve, _) => {
+        resolve({ name: 'Bob' })
+    })
+};
 
 // 5 Create a function that returns a promise which returns an array with 4 strings of 'eggs', 'apples', 'milk' and 'bread' if it resolves succesfully
 // Example:
 // returnList() resolves to the value ['eggs', 'apples', 'milk', 'bread']
 
-const returnList = () => { };
+const returnList = () => { 
+    return new Promise((resolve, _) => {
+        resolve(['eggs', 'apples', 'milk', 'bread'])
+    })
+};
 
 // 6 Create a function that returns a promise which returns "An error occurred" if it rejects
 // Example:
 // anError() rejects to "An error occurred"
 
-const anError = () => { };
+const anError = () => { 
+    return new Promise((_, reject) => {
+        reject('An error occurred')
+    })
+};
 
 // 7 Create a function that returns a promise which returns 666 if it rejects
 // Example:
 // theNumberOfTheBeast() rejects to the value 666
 
-const theNumberOfTheBeast = () => { };
+const theNumberOfTheBeast = () => {
+    return new Promise((_, reject) => {
+        reject(666)
+    })
+ };
 
 // 8 Create a function that returns a promise which returns an object with a key of error and value of 500 if it rejects
 // Example:
 // internalServerError() rejects to { error: 500 }
 
-const internalServerError = () => { };
+const internalServerError = () => {
+    return new Promise((_, reject) => {
+        reject({ error: 500 })
+    })
+ };
 
 // 9 Create a function that returns a promise which resolves to "happy" if the parameter is greater than 
 // or equal to 1 and rejects to "sad" if the value is less than 1
 // Example:
 // happySad(1) resolves to "happy"
 // happySad(0) rejects to "sad"
 
-const happySad = () => { };
+const happySad = (x) => {
```

Comment:
It's good to try to name variables so they are descriptive. Instead of using x (which doesn't tell us anything about this variable), perhaps use something like emotion of feeling (You would then have to replace x throughout this function 
```
const happySad = (emotion) => {
```